middlewares/replace_path.go

```diff
@@ -18,3 +18,8 @@ func (s *ReplacePath) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	r.URL.Path = s.Path
 	s.Handler.ServeHTTP(w, r)
 }
+
+// SetHandler sets handler
+func (s *ReplacePath) SetHandler(Handler http.Handler) {
```

Comment:
I'll go ahead and remove it.  Does that mean that it is not really needed in the other 2 places also? (`addPrefix.go` and `stripPrefix.go`)